src/mnemonic/wordlists/wordlists.md

```diff
@@ -0,0 +1,36 @@
+**Algorand Enhancement Proposal for Non-English Wordlists**
+
+We need non-English wordlist options to fast-track the adoption of Algorand and its applications.
+
+Wallets and applications like Jasiri Protocol - (https://jasiriprotocol.org) are going to take advantage of non-English wordlists to frictionlessly onboard users.
+
+A request has also been made on go-algorand #5212
+
+The English wordlist selected here remains unchanged
+All other wordlists except Arabic and Indonesian, were originally done by Bitcoin developers under BIP-39
+Arabic wordlist has been selected from https://arabeyes.org - Online Arab Linux/Unix Community
```

Comment:
Is there a more concrete link for the Arabic wordlist? I wasn't able to find the list. Similar remark regarding Indonesian.

Reply:
Here is where I sorted them from, find github repo here: https://github.com/a3f/arabic-wordlists . I used the Arabeyes wordlist, as the community there had worked on a technical dictionary already with Arabic words, see Arabeyes here: https://arabeyes.org/Download_technical_dictionary . However, as you can see from the Arabic Wordlist repo, there are many other options i.e 
hans-wehr: Hans Wehr's [Dictionary of Modern Written Arabic](https://en.wikipedia.org/wiki/Dictionary_of_Modern_Written_Arabic), English version edited by J Milton Cowan
quran: The Quran in modern Arabic writing style.
arabeyes: [Arab eyes Technical Arabic-English dictionary](http://www.arabeyes.org/Download_technical_dictionary)
arwiki: [The Arabic Wikipedia](https://ar.wikipedia.org/) on 2016-09-20. Not that useful, I guess, because of the many foreign terms and occasional typos
arzwiki: [The Egyptian-Arabic Wikipedia](https://arz.wikipedia.org/) on 2016-09-20
ar-php: ar-extra-stopwords.txt from the [Ar-PHP project](https://sourceforge.net/projects/ar-php/)

For Indonesian, here is the link: https://github.com/geovedi/indonesian-wordlist . But, as you can see, Google Code Archive shelved the links for the wordlists. I added the Indonesian wordlist for 2011 - Google Code Archive.

Thanks! @tzaffi

Reply:
I corrected and added concrete links to the files.
One quick question, do you think it is necessary to add entropy functions for the wordlists? Thanks.